### PR TITLE
New token description for NFTX Vault Index token (NFTVI)

### PR DESCRIPTION
**Pull request by admin-article21**
When submitting a new or existing set details change, make sure you have followed the following steps:

### Adding New Set Details
- [ ]  Add a new file (or update existing file) into the `set-details` or `set-details-polygon` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json). Make sure to provide the [checksum address](https://ethsum.netlify.app/) for your Set
- [ ] Verify the JSON follows the correct schema according to this [template](https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json). You can verify it by pasting the schema into this [schema validator](https://www.jsonschemavalidator.net) and checking if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (see README for example)
- [ ] If adding brand new token, make sure to add it to the `set.tokenlist.json` or `set.polygon.tokenlist.json` as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json

### Updating Existing Set Details
- [ ] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [ ] When generating a signature, make sure the version signed is for the one reflected in the version details for your entry.

### Signature (Required)

- [ ] Attach the proper signature. You can generate one by going to the page for your Set on TokenSets and signing in. Tap on your Set's name and follow the instructions provided (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

0xe7733379243deaadf41a2c62c0c8f336496b03bb3cccf6109bfc11032c0328877bab19bcbd314968e03f40454399c274130056161e10411f4a26a925c2c3c3841b

### CODE REVIEW

File: set-details/0xecf991e0d9c190d048beeb72dc39c2687f1a061e.setdetails.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "NFTX Vault Index",
+    "setSymbol": "NFTVI",
+    "setDecimals": 18,
+    "setIcon": "https://raw.githubusercontent.com/admin-article21/uniswap-tokenlist/main/assets/tokens/NFTVI_logo.png",
```
Comment (justinkchen) [inline set-details/0xecf991e0d9c190d048beeb72dc39c2687f1a061e.setdetails.json:7]: these should be referencing this repo's address (see other's for examples)

File: set-details/0xecf991e0d9c190d048beeb72dc39c2687f1a061e.setdetails.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "setDescription": "The NFTX Vault Index token (NFTVI) is the first index token of blue-chip NFTs. The token provides a diversified, market cap-weighted portfolio of the top NFT collections. Users can own fully-collateralized CryptoPunks, Bored Apes, Meebits, Squiggle, and more with a single liquid token.",
+    "managerName": "Article21",
+    "website": "https://article21.io",
+    "managerIcon": "https://raw.githubusercontent.com/admin-article21/uniswap-tokenlist/main/assets/managers/Article21_logo.png",
```
Comment (justinkchen) [inline set-details/0xecf991e0d9c190d048beeb72dc39c2687f1a061e.setdetails.json:11]: same here

**Comment by justinkchen** [comment]
also need to add to set.tokenlist.json

**Comment by ADWilkinson** [comment]
Is this still required now that you've ported your use case for use with the Index Coop?

**Comment by admin-article21** [comment]
No. Thanks Andrew. We're all set.

Cheers,
David

**Comment by justinkchen** [comment]
Is it okay to close this out then?

**Comment by admin-article21** [comment]
Yes, that's fine. Thanks for your help.

Also, thanks for noticing our budding partnership with Index!

Cheers,
J

**Pull request closed without merging**